pos_reset_search/README.rst

```diff
@@ -0,0 +1,75 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+========================================================
+Point of Sale - Reset search when user clicks on product
+========================================================
+
+The POS search workflow is to type-in the product name until there is only one product left to be selected. Then the user will select the product by typing on the "enter" key. To select a clears the search. This workflow is very efficient but it requires training.
+
+This add-on complete the POS search workflow for beginners. This add-on enables users to search and narrow down the product selection to a handful of products. The user can then use the mouse and click on the product to select. To click on the product clears search. The default workflow is still usable for trained users.
+
+This new workflow is deemed less efficient but has proven to help beginners.
+
+
+Installation
+============
+
+Normal installation.
+
+Configuration
+=============
+
+No configuration required.
+
+Limits / Roadmap
+================
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/184/9.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/{project_repo}/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed `feedback
+<https://github.com/OCA/
+pos/issues/new?body=module:%20
+pos_reset_search%0Aversion:%20
+9.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Credits
+=======
+
+Authors
+-------
+
+* Le Nid
+* CoopITEasy
```

Comment:
Nice of you but we did not do much on this code ;-)

Reply:
Me neither, I spent more time discovering the OCA PR tools than coding on this one ^^!

Reply:
Haha, first contributions are always tedious :p